src/channels/channelsDialogue.component.test.tsx

```diff
@@ -43,32 +39,60 @@ describe('selectChannelTree', () => {
             ])
           ),
         },
-        test_1: {
-          name: 'test_1',
+        Channels: {
+          name: 'Channels',
           checked: false,
           children: {
-            [testChannels[4].systemName]: {
-              ...testChannels[4],
-              checked: true,
+            '1': {
+              name: '1',
+              checked: false,
+              children: {
+                ...Object.fromEntries(
```

Comment:
I think this can be combined with a `reduce` call on `testChannels` - check if `channel.path` includes `'1'`, if so, add channel to the object.